desktop/cmp/input/JsonInput.ts

```diff
@@ -21,11 +21,18 @@ export const [JsonInput, jsonInput] = hoistCmp.withFactory<JsonInputProps>({
     displayName: 'JsonInput',
     className: 'xh-json-input',
     render(props, ref) {
+        let {value, ...jsonInputProps} = props;
+
+        if (typeof value !== 'string') {
```

Comment:
Originally was checking if value was an object, but `null` seems to be a valid value to be passed in, and is an object in JS.